server/repository/userRepository.js

```diff
@@ -21,19 +21,67 @@ class UserRepository {
   /**
  * Finds a user by email
  * @param {string} email Email to search by
+ * @param {string} username username
  * @returns {object | null} User object or null if user is not found
  */
-  static async getUserByEmail(email) {
-    const user = await User.findOne({
-      where: { email },
+  static async getUserByCredentials(email, username) {
+    const user = await User.findAll({
+      where: { email, username },
       attributes: {
-        include: ['id', 'firstName'],
-        exclude: ['createdAt', 'updatedAt', 'deletedAt', 'isDeleted', 'password']
+        include: [
+          'id',
+          'firstName'
+        ],
+        exclude: [
+          'createdAt',
+          'updatedAt',
+          'deletedAt',
+          'isDeleted',
+          'password'
+        ],
       }
     });
-    if (!user) return null;
+    if (user.length <= 0) return null;
```

Comment:
lets be consistent with our `If` statements please! No one-liners

Reply:
Okay.

Reply:
I will use a ternary operator instead

Reply:
🤦‍♂️